setup.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Yup, I've already removed the old instructions from Getting Started. I'll open an Issue to go through the other docs and fix any old references pointing to the Getting Started for installation-related instructions. Going forward, we should be consistent in pointing to the 'Flutter Setup' page as the central location for installation/configuration instructions.